uk.co.compucorp.civicrm.hrleaveandabsences/hrleaveandabsences.php

```diff
@@ -0,0 +1,238 @@
+<?php
+
+require_once 'hrleaveandabsences.civix.php';
+
+/**
+ * Implements hook_civicrm_config().
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_config
+ */
+function hrleaveandabsences_civicrm_config(&$config) {
+  _hrleaveandabsences_civix_civicrm_config($config);
+}
+
+/**
+ * Implements hook_civicrm_xmlMenu().
+ *
+ * @param array $files
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_xmlMenu
+ */
+function hrleaveandabsences_civicrm_xmlMenu(&$files) {
+  _hrleaveandabsences_civix_civicrm_xmlMenu($files);
+}
+
+/**
+ * Implements hook_civicrm_install().
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_install
+ */
+function hrleaveandabsences_civicrm_install() {
```

Comment:
Yup, sure you can add non-hook functions (see _hrleaveandabsences_civix_civicrm_xmlMenu() helper function for example).